libcontainer/state_linux.go

```diff
@@ -0,0 +1,206 @@
+// +build linux
```

Comment:
There really is not checkpointed state.  If you checkpoint, it is still running or destroyed.